### PR TITLE
Aea 884 prescription 6

### PR DESCRIPTION
**Pull request by dan-jones-iw**

### CODE REVIEW

File: models/examples/secondary-care/community/acute/nominated-pharmacy/health-professional/1-Process-Request-Send-200_OK.json [hunk 1]
```diff
@@ -465,4 +465,4 @@
       }
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (ri-ch) [inline models/examples/secondary-care/community/acute/nominated-pharmacy/health-professional/1-Process-Request-Send-200_OK.json:468]: ?

**Review by ri-ch — COMMENTED** [review]
Put the newline back in, otherwise GTG

File: models/examples/secondary-care/community/acute/nominated-pharmacy/health-professional/1-Process-Request-Send-200_OK.json (hunk 1 above)
Comment (dan-jones-iw) [reply models/examples/secondary-care/community/acute/nominated-pharmacy/health-professional/1-Process-Request-Send-200_OK.json:468]: IDE did this

Comment (dan-jones-iw) [reply models/examples/secondary-care/community/acute/nominated-pharmacy/health-professional/1-Process-Request-Send-200_OK.json:468]: this change adds the newline in

**Review by ri-ch — APPROVED** [review]

**Review by ri-ch — APPROVED** [review]

**Pull request merged**